cmd/mmc.c

```diff
@@ -1133,6 +1170,8 @@ U_BOOT_CMD(
 	"mmc partconf <dev> [[varname] | [<boot_ack> <boot_partition> <partition_access>]]\n"
 	" - Show or change the bits of the PARTITION_CONFIG field of the specified device\n"
 	"   If showing the bits, optionally store the boot_partition field into varname\n"
+	"mmc partboot dev boot_partition\n"
+	" - Return true if given boot_partition is specified in PARTITION_CONFIG, otherwise false\n"
```

Comment:
I think we don't need it anymore.
`mmc partconf <dev> varname` stores the `boot_partition` field into varname?

5c2beda545dde4d27c0e893e97c74740ff27ff2f

Reply:
You are right! Good hint - @zandrey  is just too fast ;-)